tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/PathHeader.txt

```diff
@@ -20,7 +20,11 @@
 %L = (long|short|narrow)
 %M = (Alaska_Hawaii|Bering|Dominican|Goose_Bay|Greenland_Central|Dutch_Guiana|Africa_FarWestern|Liberia|British|Irish|Kuybyshev|Sverdlovsk|Baku|Tbilisi|Turkey|Yerevan|Aktyubinsk|Ashkhabad|Dushanbe|Frunze|Kizilorda|Oral|Samarkand|Shevchenko|Tashkent|Uralsk|Urumqi|Dacca|Karachi|Borneo|Malaya|Kwajalein)
 %N = (gregorian|generic|buddhist|chinese|coptic|dangi|ethiopic|hebrew|indian|islamic|japanese|persian|roc)
-%O = (gregorian|chinese|coptic|dangi|ethiopic|hebrew|indian|islamic|persian)
+# calendar systems that should use Gregorian months (just Gregorian)
+# We don't list roc, etc here because their months are hidden.
+%G = (gregorian)
+# all others use M## form months.
+%O = (chinese|coptic|dangi|ethiopic|hebrew|indian|islamic|persian)
```

Comment:
This isn't working, I think, because what we want is that every calendar falls into one of these groups.

- Certain calendars are hidden.
- Certain calendars are non-gregorian-based, and use M1
- Certain calendars are gregorian-based, and use Jan

Because we NEVER want to miss a calendar, exactly one of these groups should be "any", and that needs to come last. That should not be the Hidden one, because we want to be safe, and better to show a calendar that we shouldn't, than hide a calendar that should be shown.

So it should be:

So
# all the calendars that should be hidden

...%XXX...

# all the calendars that should use M1

...%O...

# all the calendars that should use Jan

...%A...

Reply:
Can you detail what "isn't working"?